pom.xml

```diff
@@ -276,6 +276,77 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
Just nope 😆
Hopefully #238 should be the better fix.
Just need to figure out why one tests fails.

Reply:
Yeah, I did not expect my pom changes to make it into the merge, it is just to make it work for now :-)